pom.xml

```diff
@@ -80,10 +80,11 @@
                 <scope>import</scope>
                 <type>pom</type>
             </dependency>
+            <!-- TODO remove this after release of https://github.com/jenkinsci/workflow-step-api-plugin/pull/204 -->
```

Comment:
Well, update to the release version. Can only be removed after the BOM is updated.

Reply:
noted about bom update.

fixed comment and updated to release version